packages/block-library/src/verse/edit.js

```diff
@@ -34,6 +34,7 @@ export default function VerseEdit( {
 			</BlockControls>
 			<RichText
 				tagName={ Block.pre }
+				identifier='content'
```

Comment:
It looks like this is a required attribute to allow the verse block to be merged with the paragraph block:
https://github.com/WordPress/gutenberg/blob/4bbfbc13603d28fcc45368c529954164bf8581de/packages/block-editor/src/store/effects.js#L103
I essentially copied this from the paragraph block:
https://github.com/WordPress/gutenberg/blob/4bbfbc13603d28fcc45368c529954164bf8581de/packages/block-library/src/paragraph/edit.native.js#L39